### ISSUE TITLE
The blue "Brackets Health Report" layer does not disappear all the time

### ISSUE THREAD

**Issue by heeroluo** [adobe/brackets#11023]
The layer does not disappear although I click on it or open the "Health Report" dialog.

![brackets-health-report](https://cloud.githubusercontent.com/assets/509953/7407579/9f18197c-ef46-11e4-8b8d-8155f9ff5213.jpg)

**Comment by prafulVaishnav** [comment]
There is x button on right top to close the popup which is overridden by some extension bar. 
One way would be to disable that extension and close the popup.

**Comment by heeroluo** [comment]
Thank you, I have solved the problem. It causes by an extension called "Tabs".

**Comment by himslm01** [comment]
I also cannot close the blue "Brackets Health Report". Clicking on the X does nothing.

There is a hidden link close to the X that takes me to CHANGELOG.md in github, but no action on the X itself.

![screen shot 2015-05-23 at 16 00 08](https://cloud.githubusercontent.com/assets/1175979/7784636/ef42492e-0164-11e5-950b-3081e2b9f664.png)

**Comment by prafulVaishnav** [comment]
@himslm01 I think that must be due to some extension installed. Can you try reloading Brackets without extensions (Debug> Reload Without Extensions)

**Comment by himslm01** [comment]
@prafulVaishnav You are correct - restarting without extensions would have allowed me to press the X and also removed the hidden link to to CHANGELOG.md in github.
Without clicking on the X I updated any extensions that needed updating, and after restarting again with extensions enabled I could then click on the X. So there was an extension that needed to be updated that was causing the issue. Sorry, I did not see which extensions I updated. Sorry, I'm not being very scientific this afternoon :-(
